features/lorawan/lorastack/mac/LoRaMacCommand.cpp

```diff
@@ -93,8 +93,7 @@ void LoRaMacCommand::parse_mac_commands_to_repeat()
             case MOTE_MAC_LINK_CHECK_REQ: { // 0 byte payload
                 break;
             }
-            default:
-                break;
+            default: {}//Cannot happen
```

Comment:
I think assert should be added here as well. Default can only happen if MAC command is missing from case list.